### PR TITLE
feat: audit-logs parity with ee

### PR DESCRIPTION
**Pull request by IAmAnubhavSaini**
## Description

This PR brings in some changes that are required for Audit logs PR.

### CODE REVIEW

File: app/client/src/ce/constants/messages.ts [hunk 1]
```diff
@@ -1023,6 +1023,12 @@ export const TEST_EMAIL_FAILURE = () => "Sending Test Email Failed";
 export const DISCONNECT_AUTH_ERROR = () =>
   "Cannot disconnect the only connected authentication method.";
 export const MANDATORY_FIELDS_ERROR = () => "Mandatory fields cannot be empty";
+
+// Audit logs begin
+export const AUDIT_LOGS = () => "Audit logs";
+export const TRY_AGAIN_WITH_YOUR_FILTER = () => "Try again with your filter";
```
Comment (ankitakinger) [inline app/client/src/ce/constants/messages.ts:1029]: @IAmAnubhavSaini do we really need this in `ce/constants/messages`? I think it should be added in `ee/constants/messages`

Comment (IAmAnubhavSaini) [reply app/client/src/ce/constants/messages.ts:1029]: Will update it after EE PR gets merged. Reluctance stems from not wanting to add more changes to EE PR.

**Review by ankitakinger — APPROVED** [review]

**Comment by IAmAnubhavSaini** [comment]
https://github.com/appsmithorg/appsmith/issues/17116

**Pull request merged**